### PR TITLE
docs: minor edits for README

### PR DESCRIPTION
**Pull request by barriebyron**
remove a repeated word and share a little editorial love

## Description

closes: #XXXX

## Tasks

- [ ] 

## References

- 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by jaybxyz — APPROVED** [review]
Thanks a lot for your contribution @barriebyron. LGTM!

File: README.md [hunk 1]
```diff
@@ -25,16 +25,16 @@ cd fundraising
...
 
-These are some of the documents that help you to quickly get you on board with the fundraising module.
+These documents can help you start using the fundraising module.
 
 ## Contributing
 
-We welcome contributions from everyone. The [main](https://github.com/tendermint/fundraising/tree/main) branch contains the development version of the code. You can branch of from main and create a pull request, or maintain your own fork and submit a cross-repository pull request. If you're not sure where to start check out [CONTRIBUTING.md](./CONTRIBUTING.md) for our guidelines & policies for how we develop fundraising module. Thank you to all those who have contributed to fundraising module!
+We welcome contributions from everyone. The [main](https://github.com/tendermint/fundraising/tree/main) branch contains the development version of the code. You can branch of from main and create a pull request, or maintain your own fork and submit a cross-repository pull request. If you're not sure where to start check out [CONTRIBUTING.md](./CONTRIBUTING.md) for our guidelines and policies for how we develop fundraising module. Thank you to everyone who has contributed to fundraising module!
```
Comment (barriebyron) [inline README.md]:
```suggestion
We welcome contributions from everyone. The [main](https://github.com/tendermint/fundraising/tree/main) branch contains the development version of the code. You can branch of from main and create a pull request, or maintain your own fork and submit a cross-repository pull request. If you're not sure where to start check out [CONTRIBUTING.md](./CONTRIBUTING.md) for our guidelines and policies for how we develop the fundraising module. Thank you to everyone who has contributed to the fundraising module!
```

**Comment by barriebyron** [comment]
we need one more approver? or @kogisin can you merge? 

I am always happy to help!

**Comment by jaybxyz** [comment]
Unless it contains breaking changes, we go with one or more approvals. You can merge this PR. Thank you!

**Comment by barriebyron** [comment]
I am unable to merge @kogisin  because of permissions, I had to fork 

please merge on my behalf. If you like, you can add me to the repo

**Pull request merged**